src/main/java/io/temporal/internal/metrics/MetricsType.java

```diff
@@ -21,7 +21,6 @@
 
 public class MetricsType {
   public static final String TEMPORAL_METRICS_PREFIX = "temporal-";
```

Comment:
'-' does not work well with certain backends like prometheus for instance.  It would be nice to replace '-' with '_' everywhere if possible.  I think we have to plugin name sanitizer to fix that at runtime otherwise.